src/LibVLCSharp.WPF/ForegroundWindow.xaml.cs

```diff
@@ -58,6 +58,11 @@ void Background_Unloaded(object sender, RoutedEventArgs e)
 
         void Background_Loaded(object sender, RoutedEventArgs e)
         {
+            if (_wndhost != null)
```

Comment:
Should `_wndhost` be explicitly set to `null` at some point during the `Window` lifecycle? On `Wndhost_Closing` likely.

Reply:
We can do that, though bear in mind that the `Owner` will still remain populated if `_wndhost` is set to null, unless we choose to null that also.

The reason I used `_wndhost` as a proxy for being initialised (to ensure the logic within the `Background_Loaded` is only executed at most once) was to avoid maintaining additional state inside this class. 

If we decide to set `_wndhost` to 'null' we would also need to add a new bool field `_isInitialized` to ensure 'Background_Loaded' is only called once, because it will no longer be sufficient to check `_wndhost` for being null as a guard.

Let me know if you would like me to make this change.

Reply:
> If we decide to set _wndhost to null

I'm not clear on whether that's actually needed or not, just wondering. None of the fields are `IDisposable` so it may not be necessary. 

What do you think?

Reply:
Agreed, I don't think it's necessary.